docs/experimental-machine-learning/DGA.md

```diff
@@ -0,0 +1,92 @@
+
+You can also find some supplementary and examples [here](https://github.com/elastic/examples/tree/master/Machine%20Learning/DGA%20Detection)
+
+We also released a blog on getting started with DGA using the CLI and Kibana, which also includes a case study of the process applied to the 2020 [SolarWinds supply chain attack](https://www.elastic.co/blog/elastic-security-provides-free-and-open-protections-for-sunburst):
+* [Combining supervised and unsupervised machine learning for DGA detection](https://www.elastic.co/blog/supervised-and-unsupervised-machine-learning-for-dga-detection)
+
+
+For questions, please reach out to the ML team in the #machine-learning channel of the 
+[Elastic public slack channel](https://www.elastic.co/blog/join-our-elastic-stack-workspace-on-slack)
+
+They can also be reached by using the `stack-machine-learning` tag in the [discuss forums](https://discuss.elastic.co/tags/c/elastic-stack/stack-machine-learning)
+
+*Note: in order to use these ML features, you must have a platinum or higher [subscription](https://www.elastic.co/subscriptions)*
+*Note: the ML features are considered experimental in Kibana as well as this rules CLI*
+
+## Releases
+
+Models and dependencies will be [released](https://github.com/elastic/detection-rules/releases) as `ML-DGA-YYYMMDD-N`.
+This tag name is what will need to be passed to the CLI command.
+
+## Uploading a model and dependencies using the CLI
+
+### Usage
+
+```console
+python -m detection_rules es experimental ml setup -h
+
+Elasticsearch client:
+Options:
+  -et, --timeout INTEGER    Timeout for elasticsearch client
+  -ep, --es-password TEXT
+  -eu, --es-user TEXT
+  --cloud-id TEXT
+  --elasticsearch-url TEXT
+
+
+* experimental commands are use at your own risk and may change without warning *
+
+Usage: detection_rules es experimental ml setup [OPTIONS]
+
+  Upload ML model and dependencies to enrich data.
+
+Options:
+  -t, --model-tag TEXT       Release tag for model files staged in detection-
+                             rules (required to download files)
+  -r, --repo TEXT            GitHub repository hosting the model file releases
+                             (owner/repo)
+  -d, --model-dir DIRECTORY  Directory containing local model files
+  --overwrite                Overwrite all files if already in the stack
+  -h, --help                 Show this message and exit.
+
+```
+
+### Detailed steps
+
+#### 1. Upload and setup the model file and dependencies
+
+Run `python -m detection_rules es <args_or_config> experimental ml setup -t <release-tag>`
+
+*If updating a new model, you should first uninstall any existing models using `remove-model`*
+
+You can also upload files locally using the `-d` option, so long as the naming convention of the files match the 
+expected pattern for the filenames.
+
+#### 2. Update packetbeat configuration
+
+You will need to update your packebeat.yml config file to point to the enrichment pipeline
```

Comment:
What if I'm using Fleet?
```suggestion
You will need to update your packeybeat.yml config file to point to the enrichment pipeline
```

Reply:
Updated.

Reply:
if fleet, then you should be able to just configure the ingest pipeline in the elasticsearch index config, IINM